dotCMS/src/main/java/com/dotcms/filters/interceptor/dotcms/DefaultBackEndLoginRequiredWebInterceptor.java

```diff
@@ -0,0 +1,79 @@
+package com.dotcms.filters.interceptor.dotcms;
+
+import com.dotcms.filters.interceptor.Result;
+import com.dotcms.filters.interceptor.WebInterceptor;
+import com.dotcms.util.SecurityUtils;
+import com.dotmarketing.util.PortletURLUtil;
+import com.dotmarketing.util.SecurityLogger;
+import com.dotmarketing.util.WebKeys;
+import java.io.IOException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+
+/**
+ * @author Jonathan Gamba 9/12/18
+ */
+public class DefaultBackEndLoginRequiredWebInterceptor implements WebInterceptor {
+
+    private static final String LOGIN_URL = String
+            .format("/%s/#/public/login", PortletURLUtil.URL_ADMIN_PREFIX);
+
+    private static final String[] ALLOWED_URLS =
```

Comment:
This looks like a great start - I would say that we should offer a config property to extend this Array with custom urls